runelite-client/src/main/java/net/runelite/client/plugins/hunter/HunterPlugin.java

```diff
@@ -132,43 +134,42 @@ public void onGameObjectSpawned(GameObjectSpawned event)
 				// event runs after the player movement has been updated, so we need to
 				// compare to the trap location to the last location of the player.
 				if (lastTickLocalPlayerLocation != null
-					&& gameObject.getWorldLocation().distanceTo(lastTickLocalPlayerLocation) == 0)
+					&& trapLocation.distanceTo(lastTickLocalPlayerLocation) == 0)
 				{
 					log.debug("Trap placed by \"{}\" on {}", localPlayer.getName(), localPlayer.getWorldLocation());
-					traps.put(gameObject.getWorldLocation(), new HunterTrap(gameObject));
+					traps.put(trapLocation, new HunterTrap(gameObject));
 					lastActionTime = Instant.now();
 				}
-
 				break;
 
 			case ObjectID.NET_TRAP_9343: // Net trap placed at green sallys
 			case ObjectID.NET_TRAP: // Net trap placed at orange sallys
 			case ObjectID.NET_TRAP_8992: // Net trap placed at red sallys
 			case ObjectID.NET_TRAP_9002: // Net trap placed at black sallys
 				if (lastTickLocalPlayerLocation != null
-						&& gameObject.getWorldLocation().distanceTo(lastTickLocalPlayerLocation) == 0)
+						&& trapLocation.distanceTo(lastTickLocalPlayerLocation) == 0)
 				{
-					// Net traps facing to the south and west must have their tile translated
+					// Net traps facing to the south and west must have their tile translated.
 					// As otherwise, the wrong tile is stored.
 					Direction trapOrientation = gameObject.getOrientation().getNearestDirection();
-					WorldPoint trapLocation = gameObject.getWorldLocation();
+					WorldPoint translatedTrapLocation = trapLocation;
 
 					switch (trapOrientation)
```

Comment:
I would atleast support all orientations here, even if they are not used ingame.

Reply:
Good idea. I'll add the other orientations once I can test this again.